client/web/emberclear/app/components/pod/q-r/index.ts

```diff
@@ -1,80 +1,29 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
-import { inject as service } from '@ember/service';
 
-import { useMachine } from 'ember-statecharts';
-import { use } from 'ember-usable';
-
-import { machineConfig } from './-machine';
-import { LoginQRData } from './-types';
-
-import { SendDataConnection } from 'emberclear/services/connection/ephemeral/login/send-data';
-import CurrentUserService from 'emberclear/services/current-user';
-import { ConnectionDoesNotExistError } from 'emberclear/utils/errors';
-import QRManager from 'emberclear/services/qr-manager';
+import { StateMachine } from './state-machine';
 
 export default class QRScan extends Component {
```

Comment:
This component now no longer integrates with the XState state machine and _solely_ does component stuff